Assets/HoloToolkit/SpectatorView/UnityARKitPlugin/Plugins/iOS/UnityARKit/NativeInterface/UnityARSessionNativeInterface.cs

```diff
@@ -0,0 +1,797 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Runtime.InteropServices;
+using AOT;
+
+namespace UnityEngine.XR.iOS {
```

Comment:
Should this be in it's own folder outside of the MRTK?

Is this something devs can/should get on their own and import?

With the 2018 package manager this might make things a bit more streamlined

Reply:
We can refactor in vNext

Reply:
Looks like the ARKit plugin doesn't completely support building for UWP, so there are some changes made here that wouldn't be present if a dev imports on their own.

Reply:
> ...the ARKit plugin doesn't completely support building for UWP...

Are there issues open for the changes we've made?